pandas/core/series.py

```diff
@@ -4079,7 +4080,18 @@ def align(
             broadcast_axis=broadcast_axis,
         )
 
-    def rename(self, index=None, **kwargs):
+    def rename(
```

Comment:
The Series method doesn't completely align with the DataFrame method, but I think this is an easy way to go about it while maintain backwards compat

Reply:
What's the value in accepting `axis` here, if we don't also have `mapper` and columns?

I don't see an easy way to accept `mapper`. That's essentially fulfilled by `index`.

So maybe my question is: should we also accept `columns=`? (Not arguing for this btw. I don't know what's best here).

Reply:
Do we validate axis at all, or just ignore?